semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -414,15 +434,24 @@ def _render_group(self, semconv, parameters, output):
         self.render_ctx.is_remove_constraint = "remove_constraints" in parameters
         self.render_ctx.group_key = parameters.get("tag")
         self.render_ctx.is_full = "full" in parameters
+        self.render_ctx.is_metric_table = "metric_table" in parameters
 
         if isinstance(semconv, EventSemanticConvention):
             output.write("The event name MUST be `{}`.\n\n".format(semconv.name))
 
+        if (
+            isinstance(semconv, MetricSemanticConvention)
```

Comment:
Probably an error should be thrown if `is_metric_table` is set on another type of semantic convention.

Reply:
Good point, I've implemented that.